lib/serializer/CMemorySerializer.h

```diff
@@ -20,7 +20,7 @@ class DLL_LINKAGE CMemorySerializer
 {
 	std::vector<ui8> buffer;
 
-	size_t readPos; //index of the next byte to be read
+	size_t readPos = 0; //index of the next byte to be read
```

Comment:
Probably better to put this in constructor, to have initialization of all members in a single location.

Reply:
@IvanSavenko I can manually try put initializations into a constructor when constructor preserved, but in many cases moving init from a constructor to class can lead to removal of entire constructor.

Reply:
Perhaps we can in general remove this rule and only use it manually for small structs.
For example, in this PR `ObjectPosInfo` is the only object where in-place initialization is clearly better.

I don't think that we have large number of "dump structs", with netpacks as possible exception. And for netpacks, we can try processing it with clang-tidy separately.

Reply:
done